src/test/fuzz/hex.cpp

```diff
@@ -34,8 +32,14 @@ FUZZ_TARGET(hex)
         assert(Wtxid::FromHex(random_hex_string));
         assert(uint256::FromUserHex(random_hex_string));
     }
-    if (const auto result{uint256::FromUserHex(random_hex_string)}) {
-        assert(uint256::FromHex(result->ToString()));
+    if (auto result{uint256::FromUserHex(random_hex_string)}) {
+        auto result_string = result->ToString();
```

Comment:
nit
```suggestion
        const auto result_string{result->ToString()};
```

Reply:
Done (without the const part, which just seems like noise in this context to me, but am open to changing my mind about it)

Reply:
Again, here, I don't think the `const` is noise. This variable is passed to multiple functions, and verifying no mutation happens is an unnecessary burden on the developer. Why not just lean on the compiler?

Reply:
K, I don't feel strongly about it, pushed.